### PR TITLE
 DVCSMP-2537 Add CRC16 un-encapsulation to Z-Wave switches

### PR DESCRIPTION
**Pull request by jmagnuson**
New GE and Aeon Z-Wave switches support CRC16 encapsulation of messages.  This change adds a `Crc16Encap` event handler to support unwrapping and parsing the contained message.

Fixes DVCSMP-2663 and DVCSMP-2537.

### CODE REVIEW

**Comment by workingmonk** [comment]
cc @marstorp

**Review by varzac — APPROVED** [review]
Looks good to me.

**Review by mckeed — APPROVED** [review]
Looks good. Sorry, I thought I had already approved this.

File: devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy [hunk 1]
```diff
@@ -108,11 +108,20 @@ def updated(){
     }
 }
 
+def getCommandClassVersions() {
+	[
+		0x20: 1,  /// Basic
```
Comment (tpmanley) [inline devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy]: nit: extra `/`

**Review by tpmanley — APPROVED** [review]

**Comment by tpmanley** [comment]
Oops, my bad for merging before the squash.

**Pull request merged**